storagenode/piecestore/endpoint_test.go

```diff
@@ -233,6 +238,15 @@ func TestDownload(t *testing.T) {
 			pieceID: orderLimit.PieceId,
 			action:  pb.PieceAction_GET,
 		},
+		{ // should err with piece ID not specified
```

Comment:
If you do not define `pieceID`, won't it default to `storj.PieceID{}` as the 0 value? Is there a reason to have separate cases that set `pieceID: storj.PieceID{}` and that don't set `pieceID` at all?

Reply:
No there isn't is about me being stupidly thinking that they would be `nil`.

I'll remove them

Reply:
I think it is fine if you want to keep them to be safe. This kind of behavior confuses me still so I am happy with the extra case

Reply:
I'm not fine with them, they have duplicated test cases.
It was a mistake.